iOS/ViewControllers/Courses/LTIExerciseHintViewController.swift

```diff
@@ -62,6 +62,12 @@ class LTIExerciseHintViewController: UIViewController {
         self.itemTitleLabel.text = self.courseItem?.title
         self.launchExerciseView.isHidden = true
         self.loadingScreen.isHidden = false
+        self.loadingScreen.alpha = 0.0
+
+        // swiftlint:disable trailing_closure
+        UIView.animate(withDuration: 0.5, delay: 0.5, options: .curveLinear, animations: {
+            self.loadingScreen.alpha = 1.0
+        })
```

Comment:
You can do this once in the `LoadingViewController`. No need to repeat this code again and again 😉
Also please use `0.25` for duration and delay.

Reply:
Good point :smile: I'll fix that.